runtime/vm/jvminit.c

```diff
@@ -4379,6 +4379,25 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 			vm->extendedRuntimeFlags3 &= ~J9_EXTENDED_RUNTIME3_YIELD_PINNED_CONTINUATION;
 		}
 	}
+	{
+		IDATA argIndex = FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, VMOPT_XXUNBLOCKERTHREADWAITTIME_EQUALS, NULL);
+		if (argIndex >= 0) {
+			UDATA value = 0;
+			char *optname = VMOPT_XXUNBLOCKERTHREADWAITTIME_EQUALS;
+
+			IDATA parseError = GET_INTEGER_VALUE(argIndex, optname, value);
+			if (OPTION_OK != parseError) {
+				PORT_ACCESS_FROM_JAVAVM(vm);
+				j9nls_printf(PORTLIB, J9NLS_ERROR, J9NLS_VM_INVALID_CMD_LINE_OPT, VMOPT_XXUNBLOCKERTHREADWAITTIME_EQUALS);
+				return JNI_ERR;
+			}
+
+			vm->unblockerWaitTime= value;
+		} else {
+			/* Enable timed wait on unblocker thread by default to 64ms. */
+			vm->unblockerWaitTime= 64;
```

Comment:
same here

Reply:
Instead of use `64`, better to define a macro for this constant.

Reply:
fixed